EXAMPLES.md

```diff
@@ -0,0 +1,73 @@
+# Example
+
+This page shows some examples of converting from ssb-db (and its various helper modules, ssb-backlinks, ssb-query, etc) to ssb-db2.
+
+## Live stream of vote messages
+
+Includes also old messages.
+
+### Before
+
+```js
+pull(
+  sbot.backlinks.read({
+    query: [{ $filter: { dest: msgId } }],
+    index: 'DTA',
+    live: true
+  }),
+  pull.drain(msg => {
+    // ...
+  })
+)
+```
+
+### After
+
+```js
+const {and, votesFor, live, toPullStream} = require('ssb-db2/operators')
+
+pull(
+  sbot.db.query(
+    and(votesFor(msgId)),
```

Comment:
Hmm, my answer would be "because that's how it works" but that's not a good reason, maybe we should change the implementation so to support `votesFor(msgId)` alone